### ISSUE TITLE
Should we keep fallbacks on gradient background?

### ISSUE THREAD

**Issue by unarist** [MasterDalK/kktCSS#47]
```css
.status,
.notification,
.onboarding-modal__pager::before,
.confirmation-modal,
.boost-modal {
  /* Old browsers */
  background: rgb(252, 252, 225);
  /* FF3.6-15 */
  background: -moz-linear-gradient(top, rgba(252, 252, 225, 1) 0%, rgba(252, 252, 225, 1) 65%, rgba(252, 252, 225, 1) 65%, rgba(249, 240, 174, 1) 100%);
  /* Chrome10-25,Safari5.1-6 */
  background: -webkit-linear-gradient(top, rgba(252, 252, 225, 1) 0%, rgba(252, 252, 225, 1) 65%, rgba(252, 252, 225, 1) 65%, rgba(249, 240, 174, 1) 100%);
  /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
  background: linear-gradient(to bottom, rgba(252, 252, 225, 1) 0%, rgba(252, 252, 225, 1) 65%, rgba(252, 252, 225, 1) 65%, rgba(249, 240, 174, 1) 100%);
  /* IE6-9 */
  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#fcfce1', endColorstr='#f9f0ae', GradientType=0);
  border-radius: 8px;
  border: solid 2px #e1c152;
  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
}
```

IIRC, Mastodon doesn't seem to support Internet Explorer (incl. IE11!) and other legacy browsers. So can we get rid those polyfills?

```css
.status,
.notification,
.onboarding-modal__pager::before,
.confirmation-modal,
.boost-modal {
  background: linear-gradient(to bottom, rgba(252, 252, 225, 1) 0%, rgba(252, 252, 225, 1) 65%, rgba(252, 252, 225, 1) 65%, rgba(249, 240, 174, 1) 100%);
  border-radius: 8px;
  border: solid 2px #e1c152;
  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
}
```

**Comment by MasterDalK** [comment]
If it doesn't, then we can definitely remove them. I just use a gradient generator to make them with my photoshop files is why they exist. Does Firefox no longer need the moz gradient call?

**Comment by unarist** [comment]
Firefox supports prefix-less syntax from v16 which released 5 years ago...

See also: https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient#Browser_compatibility

**Comment by MasterDalK** [comment]
I haven't styled for Firefox before this project so I didn't know, thanks for the info though!

**Comment by MasterDalK** [comment]
Resolved in [commit f0fb130](https://github.com/MasterDalK/kktCSS/commit/f0fb1307430297641b28319a3e4ea604cd2438ac)
